### DISCUSSION TITLE
Apache is not starting from root after implementation of OIDC Proxy

### DISCUSSION THREAD

**Discussion by guptaavi75** [OpenIDC/mod_auth_openidc#1180] [General]
Hi all,

Authentication for Application is migrating from Siteminder to OIDC.
Application is using apache 2.2.29 and Oracle Linux server ver. 6.10 and it is web based application. Below dependencies are installed along with the OIDC Proxy module manually to our application.

- openssl1.0.1e-59.0.1.el6_1
- jansson-2.10
- cjose-0.6.1
- mod_auth_openidc-2.4.5 

And, we have also configured auth_openidc.conf with the required details.

But whenever we are disabling siteminder and enabling OIDC our application is not reachable and apache is not running from root.
After disabling OIDC and Siteminder both, application server is accessible and apache is running with root as well.

Please advise if we need to make any changes.

**Comment by zandbelt** [comment]
your platform (Oracle Linux 6) is only supported through a commercial support subscription via sales@openidc.om
